.pre-commit-config.yaml

```diff
@@ -38,7 +38,7 @@ repos:
     rev: v0.0.285
     hooks:
       - id: ruff
-        args: [--fix, --format, grouped, --show-fixes]
+        args: [--fix, --show-fixes]
```

Comment:
Oops, didn't mean to combine this into this PR. Though I'm a fan of removing this, the "grouped" form doesn't quick-link to the exact location in the source in terminals (like iterm) or IDEs (like VSCode and PyCharm). It's also not possible to process it into annotations (though GitHub currently has a bug keeping annotations and color output from both working together, see https://github.com/actions/runner/pull/2430).